core/src/main/java/cz/o2/proxima/scheme/StringUtf8Serializer.java

```diff
@@ -41,16 +41,41 @@ public String getDefault() {
 
     @Override
     public String asJsonValue(String value) {
-      return "\"" + value + "\"";
+      return "\"" + escape(value) + "\"";
     }
 
     @Override
     public String fromJsonValue(String json) {
       if (json.startsWith("\"") && json.endsWith("\"")) {
-        return json.substring(1, json.length() - 1);
+        return unescape(json.substring(1, json.length() - 1));
       }
       throw new IllegalArgumentException(json + "is not json string");
     }
+
+    private String escape(String value) {
```

Comment:
isn't there anything in guava / stdlib / probuf / gson we can reuse?

Reply:
maybe https://guava.dev/releases/snapshot-jre/api/docs/com/google/common/escape/Escapers.html?

Reply:
It looks like more code to use the `Escapers` than to actually escape it. :)
There is no "proper" dependency which can do json serialization in core, so I'd leave it as it is. It is covered by tests, so it should be fine.